includes/admin/plugins.php

```diff
@@ -398,6 +400,13 @@ function perflab_render_plugin_card( array $plugin_data ): void {
 			esc_html__( 'Visit plugin site', 'default' )
 		);
 	}
+
+	if ( $show_settings_link ) {
```

Comment:
Alternatively, what about getting rid of the `$show_settings_link` and just reuse `$availability['activated']`?

```suggestion
	if ( $availability['activated'] ) {
```